sway-lib-std/src/context/call_frames.sw

```diff
@@ -49,3 +57,24 @@ pub fn second_param() -> u64 {
         size: u64
     }
 }
+
+///////////////////////////////////////////////////////////
+//  Accessing arbitrary call frames by pointer
+///////////////////////////////////////////////////////////
+
+/// get a pointer to the previous (relative to the 'frame_pointer' param) call frame using offsets from a pointer.
+pub fn get_previous_frame_pointer(frame_pointer: u64) -> u64 {
+    // let offset = SAVED_REGISTERS_OFFSET + CALL_FRAME_OFFSET;
+    let offset = frame_pointer + 64 + 48;
```

Comment:
```suggestion
    let offset = SAVED_REGISTERS_OFFSET + CALL_FRAME_OFFSET;
```

Reply:
Now that we have this thing working with tests I'm happy to give it a clean up pass. I'll have to double check tomorrow, but the last time I tried to actually use these constant values here I got this:

// let offset = CODE_SIZE_OFFSET;
                              // ^^^^^^^^^^^^^^^^ Internal compiler error: Unknown variable in assembly generation. This should have been an error during type checking.Please file an issue on the repository and include the code that triggered this error.

If it persists I'll open an issue.

Reply:
We have a couple of `const` tests in the E2E, though I think the compiler complains if you don't give them a type annotation.

Reply:
This seems to be a bug. I opened an issue here: https://github.com/FuelLabs/sway/issues/1125